src/data/community-events.json

```diff
@@ -467,6 +476,15 @@
     "startDate": "2022-09-18",
     "endDate": "2022-09-25"
   },
+  {
+    "title": "LEADING PRIVACY ALLIANCE CONFERENCE",
+    "to": "https://www.leadingprivacy.com/devcon",
```

Comment:
This site still has a ton of lorem ipsum filler and default links from the WIX template. I'd prefer that was resolved before we add this event.